appcues/src/main/java/com/appcues/action/appcues/LinkAction.kt

```diff
@@ -42,10 +46,12 @@ internal class LinkAction(
 
         if (isEligibleCustomTab) {
             linkOpener.openCustomTabs(uri)
-        } else {
+        } else try {
             // this will handle any in-app deep link scheme URLs OR any web urls that were
             // requested to open into the external browser application
             appcues.navigationHandler?.navigateTo(uri) ?: linkOpener.startNewIntent(uri)
+        } catch (exception: ActivityNotFoundException) {
```

Comment:
should it catch any Exception at this level? The `ActivityNotFoundException` is specific to the `startNewIntent` implementation, but in theory, if they have a custom `navigationHandler` registered, they may throw some other Exception type and we can guard against that too?

Reply:
I would say if they do their own implementation they need to handle it themselves, at least here we can avoid generic exception

Reply:
Ok, sounds reasonable to me, thanks.